platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/SettingsTest.java

```diff
@@ -568,63 +526,8 @@ public void checkGetLoadedCryptoSubSettings() throws IOException {
     }
 
     /**
```

Comment:
This test was deleted, but the javadoc remains

Reply:
That's a bad layout of the diff.. The tests I deleted haven't had any javadoc but the test before and after. 
And some relayouting happened as well..

Reply:
Ah, I see now 👍